### PR TITLE
Adds a community folder with a Hacktoberfest page therein.

### PR DESCRIPTION
**Pull request by SuzeShardlow**

### CODE REVIEW

**Comment by SuzeShardlow** [comment]
Hi @banker and @itamarhaber, please review and merge this PR as discussed.

Unfortunately, I can't test it on my machine (the instructions in the README don't work for me) / have a Netlify preview link generated (because I need a maintainer to approve the workflow first).  Thanks!

File: community/hacktoberfest_2022.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+title: Hacktoberfest 2022 at Redis
+linkTitle: Hacktoberfest 2022
+description: Let's collaborate this October!
+weight: 1
+---
+
+[Hacktoberfest](https://hacktoberfest.com/) is a month-long online festival which takes place every year in October. It is sponsored by [DigitalOcean](https://www.digitalocean.com/) and aims to encourage people to get involved in open source projects.
+
+## How to get involved
+Through October, we're creating a number of GitHub issues for folks who want to contribute to our documentation and demo apps. [View our list of current open issues](https://github.com/search?l=&q=%23hacktoberfest+user%3Aredislabs-training+user%3Aredis+user%3Aredis-developer+user%3ANodeRedis+label%3Ahacktoberfest&state=open&type=Issues).
```
Comment (banker) [inline community/hacktoberfest_2022.md:11]:
```suggestion
Through October, we at Redis are creating a number of GitHub issues for folks who want to contribute to our documentation and demo apps. [View our list of current open issues](https://github.com/search?l=&q=%23hacktoberfest+user%3Aredislabs-training+user%3Aredis+user%3Aredis-developer+user%3ANodeRedis+label%3Ahacktoberfest&state=open&type=Issues).
```

Comment (banker) [inline community/hacktoberfest_2022.md:11]: I was able to get the link to produce issues by removing the #hacktoberfest hash tag: https://github.com/search?q=user%3Aredislabs-training+user%3Aredis+user%3Aredis-developer+user%3ANodeRedis+label%3Ahacktoberfest&type=Issues

Do we want to update this?

**Review by banker — CHANGES_REQUESTED** [review]
Looking good!

One request is that we make this into a generic Hacktoberfest page that can be reused in subsequent years. So, I'd suggest we rename the page to "hacktoberfest.md". We can update the page yearly as needed and provide information about our previous Hactoberfest participation, etc.

**Review by banker — APPROVED** [review]

**Pull request merged**